src/main.ts

```diff
@@ -105,7 +107,20 @@ async function runCargoSemverChecks(cargo: rustCore.Cargo): Promise<void> {
     // need to set the target directory explicitly.
     process.env["CARGO_TARGET_DIR"] = CARGO_TARGET_DIR;
 
-    await cargo.call(["semver-checks", "check-release"].concat(getCheckReleaseArguments()));
+    const options: any = {};
+    const handleEventData = (data: Buffer) => {
+        if (data && data.length > 0) {
+            SEMVER_CHECKS_OUTPUT += data.toString();
```

Comment:
Could we avoid making this a global, and instead keep it scoped to this function only and return it by making the function's type `Promise<string>`?

Asking because we _might_ want to run `cargo-semver-checks` more than once in a future iteration of this action (e.g. to test different combinations of features), in which case this will get cross-contaminated with the output of all the runs in a way that we won't be able to untangle easily later.

Reply:
3f610589337a8c3c91d3240288ecaa5f134b244a